### DISCUSSION TITLE
State of Workflow RFC

### DISCUSSION THREAD

**Discussion by psteinroe** [supabase/supabase#9218] [Questions] [answered]
Hey,

is there any official communication regarding the workflow feature that was teased a year ago in a [blog post](https://supabase.com/blog/supabase-workflows)? Based on recent communications, I assume that this is cancelled in favour of features that would allow larger teams to work with postgres (branching etc.). I totally approve of that decision, but it would be great to have an official statement.

**Comment by kiwicopple** [comment, marked as answer]
Hey @psteinroe - it was put on ice, simply because we realized that it would be better as a Postgres extension so that it's co-located with your data. 

The good news is that we're working on the extension prototype now, but the bad news is that I don't really have a timeline for you. We will definitely release this as a product, and 90% sure it will be a PG extension (we need to stress-test the extension before we can commit).

**Reply by psteinroe** [reply]
That's awesome, thought you abandoned it completely! Love the decision to go for a Postgres extension too. Very much looking forward to it!
